static_src/util/cf_api.js

```diff
@@ -434,11 +434,27 @@ export default {
       });
   },
 
+  putAssociateUserToEntity(userGuid, entityGuid, entityType) {
```

Comment:
need some tests to show when given the certain entityType, you eventually hit the right end point (i.e the org or space point).